### PR TITLE
Add skeleton files for exercises

### PR DESCRIPTION
**Pull request by pheanex**

### CODE REVIEW

File: exercises/linked-list/linked_list.py [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-        self.value = value
-        self.next = next
-        self.prev = prev
+class Node:
+    def __init__(self):
+        pass
 
 
-class LinkedList(object):
+class LinkedList:
```
Comment (behrtam) [inline exercises/linked-list/linked_list.py]: As we support Python 2 & 3 we should subclass from `object` to have new style classes in both versions. That goes for all other cases as well.

**Comment by behrtam** [comment]
You could use one of the available keywords (https://help.github.com/articles/closing-issues-via-commit-messages/) in your commit message to auto close issue #272 when this gets merged.

File: exercises/linked-list/linked_list.py (hunk 1 above)
Comment (pheanex) [inline exercises/linked-list/linked_list.py]: Done

**Comment by pheanex** [comment]
Ah thank you for the keyword tip. I forgot about that one :-)

**Pull request merged**